src/frontend/components/Result/Results.js

```diff
@@ -1,28 +1,37 @@
-import React from 'react';
-import './Results.css';
-import SearchBar from '../SearchBar/SearchBar';
-import Header from '../Header/Header';
-import people from '../../image/people.png';
+import React from "react";
```

Comment:
Please use a single quote in the imports as per Airbnb style guide and use double quotes just in the JSX part.